### PR TITLE
Fix labels and update select behavior

### PR DESCRIPTION
**Pull request by emeeks**
* Updates `select` inputs to reflect state
* Update labels to make them better

### CODE REVIEW

File: packages/transform-dataresource/src/index.js [hunk 1]
```diff
@@ -484,8 +507,7 @@ class DataResourceTransform extends React.Component<Props, State> {
         `}</style>
       </div>
     );
-    console.log("updatedState", updatedState);
```
Comment (jdetle) [inline packages/transform-dataresource/src/index.js:487]: Love me some console.log removals!

File: packages/transform-dataresource/src/index.js [hunk 2]
```diff
@@ -102,18 +102,29 @@ const availableLineTypes = [
...
+  selectionFunction,
+  title,
+  required,
+  selectedValue
+) => {
+  return (
+    <div style={{ display: "inline-block", margin: "0 10px" }}>
+      <h2>{title}</h2>
+      <select
+        value={selectedValue}
```
Comment (rgbkrk) [inline packages/transform-dataresource/src/index.js:116]: 😄

**Comment by rgbkrk** [comment]
Probably want to remove the changes to the notebook file here.

Thanks for making these fixes!

**Comment by emeeks** [comment]
I changed the filtering a bit on the DF to make it show two regions so that it would be a more interesting data visualization experience. But it's still smaller than the whole DF.

**Comment by rgbkrk** [comment]
Ah, cool. I'll go ahead and merge this in then.

**Pull request merged**